### PR TITLE
solution

### PR DESCRIPTION
**Pull request by AVyha**

### CODE REVIEW

**Review by mltdll — CHANGES_REQUESTED** [review]
The solution seems good, but I guess it's better to remove the images from the PR. :)

File: cinema_service/settings.py [hunk 1]
```diff
@@ -121,7 +121,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = False
```
Comment (mltdll) [inline cinema_service/settings.py:124]: Oops, missed it in the initial review. Maybe you meant `USE_TZ = False`? :)

**Review by mltdll — CHANGES_REQUESTED** [review]

File: cinema_service/settings.py (hunk 1 above)
Comment (AVyha) [reply cinema_service/settings.py:124]: 😊

**Review by mltdll — APPROVED** [review]
GJ!

**Review by Nattalli — APPROVED** [review]
👍🏻

**Pull request closed without merging**